dayliopy/clean_csv.py

```diff
@@ -27,7 +28,14 @@ def clean_row(row, default_activities):
     date = row["full_date"]
     time = row["time"]
     datetime_str = f"{date} {time}"
-    datetime_obj = datetime.datetime.strptime(datetime_str, r"%Y-%m-%d %I:%M %p")
+
```

Comment:
Can you move this logic out to a separate function? Then it means I can test it easier and make a unit test.

You could call like this

```python
datetime_str = f"{date} {time}"
datetime_obj = parse_datetime(datetime_str)

# Match the mood
# ...
```